doc/api/v8.md

```diff
@@ -356,6 +356,18 @@ if (isMainThread) {
 }
 ```
 
+## `v8.setHeapSnapshotNearHeapLimit(limit)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+The API is a no-op if `--heapsnapshot-near-heap-limit` is already set from the
```

Comment:
Would you mind explaining why this API can not be called if the flag is set from the commandline or can not be called twice?

Reply:
I have update the code and make the API can be called twice, please help review again, thanks !